extensions/adobe/experience/target/classification.schema.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
If you are intending this to be a class then you need to add
`"meta:extends": [
    "https://ns.adobe.com/xdm/data/record"
  ],`
If you are intending this to be a field group then you need to add
`"meta:intendedToExtend": [
    "https://ns.adobe.com/xdm/data/record"
  ],`